sdl-deltalake/src/main/scala/io/smartdatalake/workflow/dataobject/DeltaLakeTableDataObject.scala

```diff
@@ -199,17 +200,65 @@ case class DeltaLakeTableDataObject(override val id: DataObjectId,
     }
   }
 
+  private def activateCdc()(implicit context: ActionPipelineContext): Unit = {
+    implicit val session: SparkSession = context.sparkSession
+    if(!propertyExists(EnableCdcFeedProperty) && isTableExisting) HiveUtil.alterTableProperties(table, Map(EnableCdcFeedProperty -> "true"))
+  }
+
+  private def propertyExists(name: String)(implicit session: SparkSession): Boolean = {
+    val details = DeltaTable.forName(session, table.fullName).detail()
+    val properties = details.select("properties").head.getMap[String, String](0)
+
+    properties.contains(name)
+  }
+
+  private def propertyExistsWithValue(name: String, value: String) (implicit session: SparkSession): Boolean = {
+    val details = DeltaTable.forName(session, table.fullName).detail()
+    val properties = details.select("properties").head.getMap[String, String](0)
+
+    properties.exists(_ == name -> value)
+  }
+
+  @transient private val EnableCdcFeedProperty = "delta.enableChangeDataFeed"
```

Comment:
Variable names should start with lowercase:
```suggestion
  @transient private val enableCdcFeedProperty = "delta.enableChangeDataFeed"
```